server/webpa.go

```diff
@@ -81,6 +83,8 @@ type Basic struct {
 	CertificateFile    string
 	KeyFile            string
 	ClientCACertFile   string
+	PeerVerify	   bool
+	PeerVerifyFunc     PeerVerifyCallback	// Callback func to add peer client cert CN, SAN validation
```

Comment:
The zero value for this variable should be valid.  Create a default verify callback that is used when this variable is `nil` and `PeerVerify` is `true`